test/game_reader_test.rb

```diff
@@ -6,4 +6,45 @@
 
 class GameReaderTest < Minitest::Test
 
+  def test_it_communicates_with_scrabble
```

Comment:
I really like these tests, but make sure that you can run them from the project directory. It seems like you're using a combination of require/require_relative, but not running from the project directory. When I pulled down and tried to run from the project directory, these wouldn't run.